lib/storage/src/content_manager/toc.rs

```diff
@@ -381,10 +381,11 @@ impl TableOfContent {
         }
     }
 
+    // lifetime required by rustc as Rust 1.62
```

Comment:
that piece of code start yielding rustc or clippy errors when I add my function underneath :man_shrugging:

Reply:
I don't see any lifetime related changes [there](https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1620-2022-06-30) and lifetimes now look a bit bizarre. Do you think we should try to submit a bug to rustc?